src/store/settingsSlice.ts

```diff
@@ -33,6 +44,14 @@ const initialState: SettingsState = {
     qr: true,
   },
   theme: {},
+  env: {
+    cgw: '',
+    rpc: {},
+    tenderly: {
+      url: '',
+      accessToken: '',
+    },
+  },
```

Comment:
```suggestion
 env:
    | {
        cgw?: string
        tenderly: {
          url?: string
          accessToken?: string
        }
        rpc: {
          [chainId: string]: string
        }
      }
    | undefined

Reply:
When running the branch locally, we get this error
![Screenshot 2023-01-16 at 10 22 43](https://user-images.githubusercontent.com/32431609/212646374-3e93e822-8201-490c-b5fc-00e7b42a2195.png)
because https://github.com/safe-global/web-core/blob/dev/src/store/storeHydrator.tsx overwrites the settings redux slice with the persisted state

Reply:
You can inspire from when `hiddenTokens` was added to this redux slice

Reply:
Good catch! I adjusted the type to fix this.